mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelFactory.kt

```diff
@@ -101,11 +101,13 @@ data class FragmentViewModelContext(
     /**
      * The fragment owner of the ViewModel.
      */
-    val fragment: Fragment
+    val fragment: Fragment,
+    private val ownerProvider: ViewModelStoreOwner = fragment,
+    private val savedStateRegistryProvider: SavedStateRegistry = fragment.savedStateRegistry
```

Comment:
Why not just override the vals in the constructor? Can you also add some docs for why somebody would want to use this?

Reply:
> Why not just override the vals in the constructor?

The `FragmentViewModelContext` constructor? They're not exposed tho https://github.com/airbnb/MvRx/blob/66469dc2e66e03a129fc30766f458e9648cbc68c/mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelFactory.kt#L98-L105

Sure I can add some docs. For my current use case, when you use Jetpack Navigation Component you can use the navigation graph ViewModel stores instead of the fragments store. I need a different store then just the fragment I need something like this 
https://github.com/airbnb/MvRx/blob/d8141ff9812a6f00cccf48d89c181a2c13e261df/mvrx-navigation/src/main/kotlin/com/airbnb/mvrx/navigation/MvRxExtensions.kt#L18-L37

Reply:
@marukami Makes sense but it's best to make the docs ultra clear so other people can understand too :)

Reply:
@gpeal No worries, updated class docs.